mayastor/src/pool.rs

```diff
@@ -355,7 +405,55 @@ impl Iterator for PoolsIter {
     }
 }
 
-fn list_pools() -> Vec<jsondata::Pool> {
+pub async fn create_pool(args: rpc::mayastor::CreatePoolRequest)
```

Comment:
Some functions are pub some of them are pub (crate) let us try to use pub (crate) everywhere where possible.

Reply:
OK, I have tried to make it more consistent.